tests/CommunityHydration-test.js

```diff
@@ -0,0 +1,50 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const CommunityHydration = require('../src/CommunityHydration');
+const Hydration = require('../src/Hydration');
+
+describe('CommunityHydration', function() {
+  let communityHydration, hydration;
+
+  beforeEach(function() {
+    communityHydration = new CommunityHydration([
```

Comment:
Really thorough test cases, good call. I'm need to add more to mine.

Reply:
@srslie Thank you!